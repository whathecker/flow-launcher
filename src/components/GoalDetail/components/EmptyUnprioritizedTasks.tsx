import React from "react";
import { StyleSheet } from "react-native";
import { View, Text } from "../../Themed";
import { Container, Typography, Color } from "../../../styles";

const EmptyUnprioritizedTasks: React.FC = () => {
  return (
    <View style={styles.emptyTasksMsgWrapper}>
      <Text style={styles.emptyTasksMsg}>{"Nothing there yet"}</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  emptyTasksMsgWrapper: {
    ...Container.centerAligned,
    height: "55%",
    padding: 20,
  },
  emptyTasksMsg: {
    ...Typography.p,
    fontSize: 16,
    color: Color.light.subtleLabel,
  },
});

export default EmptyUnprioritizedTasks;
