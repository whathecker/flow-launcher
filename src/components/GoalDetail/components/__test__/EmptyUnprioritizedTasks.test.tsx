import * as React from "react";
import renderer from "react-test-renderer";
import EmptyUnprioritizedTasks from "../EmptyUnprioritizedTasks";

describe("Test EmptyUnprioritizedTasks component", () => {
  it("Should renders EmptyUnprioritizedTasks correctly", () => {
    const tree = renderer.create(<EmptyUnprioritizedTasks />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
