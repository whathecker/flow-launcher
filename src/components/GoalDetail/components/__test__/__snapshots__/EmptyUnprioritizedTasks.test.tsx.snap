// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test EmptyUnprioritizedTasks component Should renders EmptyUnprioritizedTasks correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
      },
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "height": "55%",
        "justifyContent": "center",
        "padding": 20,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#000000",
        },
        Object {
          "color": "#848484",
          "fontFamily": "Lexend-Regular",
          "fontSize": 14,
        },
      ]
    }
  >
    Nothing there yet
  </Text>
</View>
`;
